source/declarations.tex

```diff
@@ -5475,7 +5475,7 @@
 reference-compatible with ``\cvqual{cv3} \tcode{T3}'' (see~\ref{over.match.ref}),
 \end{itemize}
 then
-the value of the initializer expression in the first case and
+the initializer expression in the first case and
 the result of the conversion in the second case
```

Comment:
I think the second case is also wrong. Suggestion:
```suggestion
the converted expression in the second case
```
A "result" of an expression is a value or object, and can't be a prvalue as required in the next sentence.